client/network/test/src/sync.rs

```diff
@@ -285,13 +284,13 @@ fn sync_justifications() {
 	runtime.block_on(futures::future::poll_fn::<(), _>(|cx| {
 		net.poll(cx);
 
-		for hash in [hashof10, hashof15, hashof20] {
-			if net.peer(0).client().justifications(hash).unwrap() !=
+		for height in (10..21).step_by(5) {
```

Comment:
This was just reverted to original form:
https://github.com/paritytech/substrate/blob/97ad64bd507eacf1dc8c597cf8538c98facb3220/client/network/test/src/sync.rs#L280-L291